docs/deployment_patterns/how_to_use_gx_with_aws/how_to_use_gx_with_aws_using_cloud_storage_and_pandas.md

```diff
@@ -0,0 +1,325 @@
+
+<!-- Part 1: Setup -->
+
+<!-- 1.1 Ensure that the AWS CLI is ready for use -->
+
+<!-- 1.1.1 Verify that the AWS CLI is installed -->
+import VerifyAwsInstalled from './components/_aws_cli_verify_installation.md'
+
+<!-- 1.1.2 Verify that your AWS credentials are properly configured -->
+import VerifyAwsCredentials from '@site/docs/guides/setup/configuring_metadata_stores/components/_verify_aws_credentials_are_configured_properly.mdx'
+
+<!-- 1.2 Prepare a local installation of Great Expectations -->
+
+<!-- 1.2.1 Verify that your Python version meets requirements -->
+
+import VerifyPythonVersion from '@site/docs/guides/setup/installation/components_local/_check_python_version.mdx'
+import WhereToGetPython from './components/_python_where_to_get.md'
+
+<!-- 1.2.2 Create a virtual environment for your Great Expectations project -->
+
+import CreateVirtualEnvironment from '@site/docs/guides/setup/installation/components_local/_create_an_venv_with_pip.mdx'
+
+<!-- 1.2.3 Ensure you have the latest version of pip -->
+
+import GetLatestPip from '@site/docs/guides/setup/installation/components_local/_ensure_latest_pip.mdx'
+
+<!-- 1.2.4 Install boto3 -->
+
+import InstallBoto3WithPip from '@site/docs/guides/setup/configuring_metadata_stores/components/_install_boto3_with_pip.mdx'
+
+<!-- 1.2.5 Install Great Expectations -->
+
+import InstallGxWithPip from '@site/docs/guides/setup/installation/components_local/_install_ge_with_pip.mdx'
+
+<!-- 1.2.6 Verify that Great Expectations installed successfully -->
+
+import VerifySuccessfulGxInstallation from '@site/docs/guides/setup/installation/components_local/_verify_ge_install_succeeded.mdx'
+
+<!-- 1.3 Create your Data Context -->
+
+import CreateDataContextWithCli from '@site/docs/guides/setup/configuring_data_contexts/components_how_to_configure_a_new_data_context_with_the_cli/_initialize_data_context_with_the_cli.mdx'
+
+<!-- 1.4 Configure your Expectations Store on Amazon S3 -->
+
+<!-- 1.4.1 Identify your Data Context Expectations Store -->
+
+import IdentifyDataContextExpectationsStore from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_identify_your_data_context_expectations_store.mdx'
+
+<!-- 1.4.2 Update your configuration file to include a new Store for Expectations on Amazon S3 -->
+
+import AddS3ExpectationsStoreConfiguration from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_update_your_configuration_file_to_include_a_new_store_for_expectations_on_s.mdx'
+
+<!-- 1.4.3 Verify that the new Amazon S3 Expectations Store has been added successfully -->
+
+import VerifyS3ExpectationsStoreExists from '@site/docs/guides/setup/configuring_metadata_stores/components_how_to_configure_an_expectation_store_in_amazon_s3/_confirm_that_the_new_expectations_store_has_been_added_by_running_great_expectations_store_list.mdx'
```

Comment:
For both the `ExpectationStore` and `ValidationStore` steps, I wasn't able to list the previous store (Non-S3) when running `great_expectations store list`, when the document says : "Notice the output contains two Expectation Stores..." 

Not sure if this is just for me, or if the CLI behavior has changed here.